### DISCUSSION TITLE
Is it possible to reference values from query in an alert?

### DISCUSSION THREAD

**Discussion by danfinn** [grafana/grafana#68903] [Alerting]
I have an alert that's checking an elasticsearch datasource for kubernetes events that are warning.  It would make the event much more useful if I could include some fields from the query to add context to the alert.  Is this possible?

**Comment by danfinn** [comment]
or if that is not possible is it possible to include the name of the datasource that the alert came from?  this alert checks multiple datasources for errors

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
